UPGRADING.md

```diff
@@ -2,6 +2,10 @@
 
 This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
 
+## v10
+
+Dropped node 9.
```

Comment:
Do we really need to bump major when dropping node version with an odd version number?

Reply:
If you're strict about semver, it's a breaking (major) change. If I understand, node 9 is dropped, but it's been EOL since [July 2017](https://github.com/nodejs/Release#lts_schedule). Perhaps we can keep it until there's another breaking change (more worthwhile).

Reply:
I prefer a simple policy. Drop a node version, bump major. Where's the harm?

Reply:
Nevermind, I just saw @ralphtheninja's comment in another thread, making the point that we do still support the even versions surrounding 9.

In that case, bumping is a mistake, will revert